rlay-client-lib/src/client.js

```diff
@@ -0,0 +1,107 @@
+const Web3 = require('web3');
+const pLimit = require('p-limit');
+const rlay = require('@rlay/web3-rlay');
+const EntityMetaFactory = require('./entity/meta-factory');
+const RlayEntities = require('./rlay');
+
+class Config {
+  constructor() {
+    this.address = '0xc02345a911471fd46c47c4d3c2e5c85f5ae93d13';
+    this.backed = 'myneo4j';
```

Comment:
`backed` -> `backend`

This causes an RPC error message when trying to use it.